client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:16-alpine 
+WORKDIR /app
+COPY . .
```

Comment:
Not everything should be copied into the image here, this this will cause cache busting for the dependancies any time there is an application change